### ISSUE TITLE
[REQUEST] Please make a flathub version for flatpak integration

### ISSUE THREAD

**Issue by asif-hoda** [vinceliuice/WhiteSur-gtk-theme#130]
None of my flatpaks follow your theme, they default to adawaita. So please make flatpak for this gtk theme

**Comment by vinceliuice** [comment]
Just run this:

sudo flatpak override --filesystem=~/.themes

**Comment by asif-hoda** [comment]
It did not work

**Comment by asif-hoda** [comment]
Interesting..... the theme is working for every flatpak except gimp. Why is gimp so wierd. I use it so much. Please help me fix its theme. It just feels so disconnected to everything

**Comment by vinceliuice** [comment]
Isn't gimp set the theme in its own settings?

**Comment by asif-hoda** [comment]
Yeah from there you just have to select system theme as an option, but whenever i do that adawaita comes up instead of my default theme

**Comment by CyberAzure** [comment]
That's strange...maybe a GIMP issue? a Flatpak issue?
